en/frames_rover/differential.md

```diff
@@ -0,0 +1,11 @@
+# Differential Rovers
+
```

Comment:
FYI only, normally I am reticent to change file names, delete files etc. The reason is that there is no auto-redirects so any linking to docs will be broken. This doesn't matter so much for this doc because it hasn't been in a previous version, or at least was not very stable there. But for anything that did pre-exist we should _consider_ a redirect. 

For example, when I split the flight mode docs into separate folders for easier management I created docs like this to redirect https://raw.githubusercontent.com/PX4/PX4-user_guide/refs/heads/main/en/flight_modes/altitude_fw.md .
Eventually I might delete these.

Anyway, "as a habit" consider whether the name change is worth the cost. For now we're finding a new structure and this is good.